Content.Server/Station/Systems/StationSpawningSystem.cs

```diff
@@ -69,7 +74,17 @@ public override void Initialize()
         if (station != null && !Resolve(station.Value, ref stationSpawning))
             throw new ArgumentException("Tried to use a non-station entity as a station!", nameof(station));
 
+        var prioritizeArrivals = _configurationManager.GetCVar(CCVars.StationSpawningPrioritizeArrivals);
         var ev = new PlayerSpawningEvent(job, profile, station);
+
+        if (station != null && prioritizeArrivals)
+            _arrivalsSystem.HandlePlayerSpawning(ev);
+
+        _containerSpawnPointSystem.HandlePlayerSpawning(ev);
+
+        if (station != null && !prioritizeArrivals)
+            _arrivalsSystem.HandlePlayerSpawning(ev);
```

Comment:
Looks cleaner to just if-else as this control flow is weird.

Also isn't this supposed to be a client cvar.

Reply:
Ahh okay, that's what they meant by customization.